code/modules/cm_aliens/structures/fruit.dm

```diff
@@ -283,11 +289,34 @@
 		return XENO_NO_DELAY_ACTION
 	return ..()
 
-/obj/effect/alien/resin/fruit/speed/consume_effect(mob/living/carbon/Xenomorph/recipient)
+/obj/effect/alien/resin/fruit/speed/consume_effect(mob/living/carbon/Xenomorph/recipient, var/do_consume = TRUE)
 	if(mature && recipient && !QDELETED(recipient))
 		to_chat(recipient, SPAN_XENONOTICE("The [name] invigorates you to move faster!"))
 		new /datum/effects/xeno_speed(recipient, ttl = speed_duration, set_speed_modifier = speed_buff_amount, set_modifier_source = XENO_FRUIT_SPEED, set_end_message = SPAN_XENONOTICE("You feel the effects of the [name] wane..."))
-	finish_consume(recipient)
+	if(do_consume)
+		finish_consume(recipient)
+
+/obj/effect/alien/resin/fruit/plasma
+	name = XENO_FRUIT_PLASMA
+	desc = "A fruit that can be eaten to boost your plasma generation."
+	time_to_mature = 25 SECONDS
+	icon_state = "fruit_plasma_immature"
+	mature_icon_state = "fruit_plasma"
+	consumed_icon_state = "fruit_spent_2"
+	flags = CAN_CONSUME_AT_FULL_HEALTH
+	fruit_type = /obj/item/reagent_container/food/snacks/resin_fruit/plasma
+	glow_color = "#287A90"
+	var/plasma_amount = 240
+	var/plasma_time = 15
+	var/time_between_plasmas = 3
+
+/obj/effect/alien/resin/fruit/plasma/consume_effect(mob/living/carbon/Xenomorph/recipient, var/do_consume = TRUE)
+	if(mature && recipient && recipient.plasma_max > 0 && !QDELETED(recipient))
+		to_chat(recipient, SPAN_XENONOTICE("The [name] boosts your plasma regeneration!"))
```

Comment:
```suggestion
		to_chat(recipient, SPAN_XENONOTICE("\The [name] boosts your plasma regeneration!"))
```

Reply:
[name] is a string, not a mob, so \the wouldn't work with it

Reply:
wah